fr.perl6intro.adoc

```diff
@@ -2297,22 +2426,25 @@ if "Albert-Ebasque" ~~ / <:Pd> / {
 ----
 
 === Métacaractères
-Les métacaractères (Wildcards) peuvent également être utilisées dans les regex.
+Dans une regex, tout caractère ni alphanumérique, ni tiret-bas est potentiellement un métacaractère,
+c'est à dire qu'il n'est pas interprété de manière littérale.
+Comme on l'a déjà vu, les métacaractères doivent être échappés pour être inteprétés de manière
+littérale.
 
 Le point `.` signifie n'importe quel caractère unique.
 
 [source,perl6]
 ----
 if 'abc' ~~ m/ a.c / {
-    say "Correspondance";
+    say "match";
```

Comment:
Quand les mots anglais sont courants et courts, je les préfère aux mot
français correspondants dont l'usage n'est pas établi pour la traduction ou
qui sont ambigus. Ainsi je ne traduis pas hash en table de hachage alors
que les livres O'Reilly le font. Puisque c'est du Perl, j'aime huffmnaniser
pour faire court.
S'il est plus facile d'apprendre un langage dans sa langue maternelle, il
est de toute façon utile de connaître les termes techniques en anglais.

Bon courage pour ta traduction.

On Thursday, September 1, 2016, Breno G. de Oliveira <
notifications@github.com> wrote:

> In fr.perl6intro.adoc
> <https://github.com/hankache/perl6intro/pull/35#discussion_r77110687>:
>
> >
> >  Le point `.` signifie n'importe quel caractère unique.
> >
> >  [source,perl6]
> >  ----
> >  if 'abc' ~~ m/ a.c / {
> > -    say "Correspondance";
> > +    say "match";
>
> Pardon, mais pourquoi changer "correspondance" avec "match"? Je demande
> parce que je ne sais pas si "match" devrait être traduit en portugais aussi
> (désolé pour mes terribles français).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hankache/perl6intro/pull/35/files/8fa437a397ff1c36d6d7e67df3df1c2fcf455af9#r77110687>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAA_MHnDs75McZvGZUmBGgZDEosG6Xm9ks5qlkwZgaJpZM4HrqkX>
> .
>


-- 
cognominal stef